packages/@extension-kid/notification-service/__tests__/NotificationService-test.ts

```diff
@@ -0,0 +1,85 @@
+import { Container } from "@extension-kid/core";
+import notificationServiceContainerModuleFactory, {
+  getExtensionModule,
+  Notification,
+  NotificationService,
+  NotificationServiceType,
+  NotificationServiceExtensionInterface
+} from "../index";
+import { injectable } from "inversify";
+
+const DebugNotification = Symbol("DebugNotification");
+
+const DebugSymbol = Symbol("Debug");
+
+@injectable()
+class DebugNotificationExtension
+  implements NotificationServiceExtensionInterface {
+  id = DebugSymbol;
+
+  supportedNotifications(): symbol[] {
+    return [DebugNotification];
+  }
+
+  push(notification: Notification) {
+    /* tslint:disable */
+    console.log(
+      `${notification.timestamp.toISOString()} - DEBUG - ${
+        notification.message
+      }`
+    );
+    /* tslint:enable */
+  }
+}
+
+describe("NotificationService", () => {
+  let container: Container;
+  beforeEach(() => {
+    container = new Container();
+
+    container.load(notificationServiceContainerModuleFactory());
+  });
+
+  it("can get Notification service", () => {
+    const ns: NotificationService = container.get<NotificationService>(
```

Comment:
we only need one of those annotations:

```
const ns: NotificationService = container.get<NotificationService>(
          ^^^^^ EITHER ^^^^^^                 ^^^^^^^ OR ^^^^^^^^
```

the other is inferred automatically